pyccl/halos/massdef.py

```diff
@@ -0,0 +1,247 @@
+from .. import ccllib as lib
+from ..core import check
+from ..background import species_types, rho_x, omega_x
+import numpy as np
+
+
+def mass2radius_lagrangian(cosmo, M):
+    """ Returns Lagrangian radius for a halo of mass M.
+    The lagrangian radius is defined as that enclosing
+    the mass of the halo assuming a homogeneous Universe.
+
+    Args:
+        cosmo (:obj:`Cosmology`): A Cosmology object.
+        M (float or array_like): halo mass in units of M_sun.
+
+    Returns:
+        float or array_like: lagrangian radius in Mpc
+    """
```

Comment:
The use of lagrangian here confuses me. Also is the output in comoving units? I think yes.

Reply:
I think the nomenclature is standard.
https://bdiemer.bitbucket.io/colossus/lss_peaks.html#lss.peaks.lagrangianR
(i.e. it's the comoving radius of the halo in the initial condition).
Added comoving